www/proxy-sso/src/opnsense/scripts/OPNsense/ProxySSO/kerberos_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+PASS_TMP=/tmp/__tmp_kerb_pass
+
+while getopts :f:u:p: name
+do
+    case $name in
+    f) FQDN="$OPTARG" ;;		# aka TING.tingnet.local
+    u) USERNAME="$OPTARG" ;;		# username
+    p) PASSWORD="$OPTARG" ;;		# password
+    esac
+done
+
+[ "$USERNAME" == "" ] && echo "No account name" && exit 0;
+[ "$PASSWORD" == "" ] && echo "No account password" && exit 0;
+[ "$FQDN" == "" ] && echo "No FQDN" && exit 0;
+
+PASSWORD="${PASSWORD%\'}"
+echo "${PASSWORD}" | sed 's/\\//g' > ${PASS_TMP}
+
+/usr/local/bin/kinit ${USERNAME} < ${PASS_TMP}
```

Comment:
is that safe against RCE?

Reply:
I think so.
Password escaped before this action call:
```
$pass = escapeshellarg($this->request->getPost("password"));
```
 And additional quoting performed in script (%\\').